tests/mail/tests.py

```diff
@@ -604,6 +605,35 @@ def test_sanitize_address(self):
             '=?utf-8?q?T=C3=B3_Example?= <=?utf-8?b?dMOz?=@example.com>'
         )
 
+@unittest.skipIf(sys.platform.startswith('win'), "Windows tests would require non-UTC Jenkins server.")
```

Comment:
I guess you probably want `from django.test.utils import requires_tz_support` instead.